### PR TITLE
Add !, #hell and $_$

### PR DESCRIPTION
**Pull request by Abelkrijgtalles**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

#### Link to programming language: 
!: https://github.com/Dylan-Middendorf/Negation, https://esolangs.org/wiki/!
#hell: https://esolangs.org/wiki/HashHell
$_$: https://github.com/Sxakalo/SUS, https://esolangs.org/wiki/$_$

### CODE REVIEW

**Review by Richienb — CHANGES_REQUESTED** [review]
> The code displays "Hello World"
Does not appear to be the case

**Comment by Abelkrijgtalles** [comment]
Do you mean by $_$? Because that Is not something like
`print("Hello world")`

**Comment by Richienb** [comment]
> Do you mean by $_$? Because that Is not something like `print("Hello world")`

Yes. Changing it appears to be easy though since each letter seems to be seperated by an ampersand. https://esolangs.org/wiki/$_$

File: #/#hell [hunk 1]
```diff
@@ -0,0 +1 @@
+io.write("Hello, World!")
```
Comment (MrBrain295) [inline #/#hell:1]: This needs to be “Hello World”.

**Comment by MrBrain295** [comment]
@Abelkrijgtalles why did you close this?

**Pull request closed without merging**